std-bits/base/src/main/java/org/enso/base/enso_cloud/logging/LogMessage.java

```diff
@@ -0,0 +1,102 @@
+package org.enso.base.enso_cloud.logging;
+
+import com.fasterxml.jackson.databind.node.JsonNodeFactory;
+import com.fasterxml.jackson.databind.node.NullNode;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.fasterxml.jackson.databind.node.TextNode;
+import java.util.Objects;
+import org.enso.base.CurrentEnsoProject;
+import org.enso.base.enso_cloud.CloudAPI;
+
+/** Base class for log messages that are passed to the OpenSearch cloud endpoint. */
```

Comment:
```suggestion
/** Base class for log messages that are passed to the cloud endpoint. */
```

No need to mention that

Reply:
Done in 00d90029d6eb00390328e0d50035a1926ec26973